presto-native-execution/presto_cpp/main/types/tests/TypeSignatureTest.cpp

```diff
@@ -400,8 +400,12 @@ TEST_F(TestTypeSignature, functionType) {
 }
 
 TEST_F(TestTypeSignature, decimalType) {
-  assertSignature("decimal(10, 5)", "DECIMAL(10,5)");
-  assertSignature("decimal(20,10)", "DECIMAL(20,10)");
+  ASSERT_TRUE(
+      parseTypeSignature("decimal(10, 5)")->toString() == "DECIMAL(10, 5)" ||
```

Comment:
Consider extracting local variable for readability and adding a comment to explain the logic.

```
// https://github.com/facebookincubator/velox/pull/6295 changes the output of Type::toString() in Velox to add a space after comma. To avoid breakages, temporarily, allow both formats.

auto signature = parseTypeSignature("decimal(10, 5)")->toString();
ASSERT_TRUE(signature == "DECIMAL(10, 5)" || signature == "DECIMAL(10,5)");
...
```